test/unit/connection/connection_config_test.js

```diff
@@ -721,6 +835,20 @@ describe('ConnectionConfig: basic', function ()
           username: 'username',
           password: 'password'
         }
+      },
+      {
+        name: 'oauthwithoutusername',
```

Comment:
Can we added spaces as in other test names?

Reply:
updated